### PR TITLE
use StringUtils's isNotBlank and isBlank instead of trim().length() or trim().isEmpty() etc

### PR DESCRIPTION
**Pull request by XenoAmess**
…

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -141,6 +141,7 @@ subprojects {
     version = '5.2.3-SNAPSHOT'
     ext {
         bouncyCastleVersion = '1.70'
+        commonsLang3Version = '3.12.0'
```
Comment (pjfanning) [inline build.gradle]: This appears in another PR you added and again, this does not seem important enough to add another dependency.

Comment (XenoAmess) [reply build.gradle]: Or you can just copy the functions needed, means isBlank and isNotBlank, from commons-lang3, which is an apache v2 licensed repo from apache, so get some codes from there seems resonable, as this is also an apache v2 licensed repo from apache

Comment (pjfanning) [reply build.gradle]: POI has its own StringUtil class - you can add extra methods there.

**Comment by XenoAmess** [comment]
@pjfanning done. please review again.

**Comment by pjfanning** [comment]
thanks - merged

**Pull request closed without merging**